OneSchedule/ExceptionHandlingMiddleware/ExceptionHandlingMiddleware.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System.Text.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Telegram.Bot.Types;
+using System.IO;
+using Telegram.Bot;
+using OneSchedule.Settings;
+using Microsoft.Extensions.Options;
+
+namespace OneSchedule
+{
+    public class ExceptionHandlingMiddleware
+    {
+        private readonly RequestDelegate _next;
+        private ILogger<ExceptionHandlingMiddleware> _logger;
+        private ITelegramBotClient _bot;
+
+        public ExceptionHandlingMiddleware(RequestDelegate next, ILogger<ExceptionHandlingMiddleware> logger,
```

Comment:
Format code properly, see this link, first chapter https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions